sound/sequences/00_sound_player.s

```diff
@@ -2579,7 +2591,7 @@ chan_setlayer 1, .layer_F18
 chan_end
 
 .layer_F18:
-layer_note1 39, 0x46, 127
+layer_note1_voice 39, 0x46, 127
```

Comment:
not sure about extending this to peach's sound effects since i think its much more common for people to only want to replace mario's sfx. doing it like this pretty much ruins the ending credits and people might not notice this

Reply:
I added these for consistency; in practice, these don't actually do anything different with the define enabled.

Reply:
ah yeah i noticed they're 39 anyway, well i still think that it's not really "consistency" since i think mario's sfx are already a well-defined enough subset

Reply:
I wanted to pretty much tie the macros to the channel bank as a whole since I think that just makes more sense in context.

Reply:
still funny merge conflict potential - seeing as the point of the defines is so that people don't have to touch seq00 to begin with, in my opinion minimizing changes is the best

Reply:
I don't see it tbh; if anything that's especially likely to not happen with Peach sounds compared to Mario (plus this is 2.1 scope so conflicts are of a lesser concern to me)

Reply:
its certainly not a huge downside but i honestly cant see any amount of upside either

Reply:
Ehhhh fine, I'll change the macros to `_mario` and remove them from Peach stuff if you really insist. Note I'm only giving in here because people replace Peach samples with non-voice sfx all the time lmao

Reply:
lmao yeah i did that myself for mario olympics

Reply:
done